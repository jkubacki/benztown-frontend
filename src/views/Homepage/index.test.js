import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { HomepageUnwrapped as Homepage } from './index';

const props = { isLoggedIn: false };

it('renders without crashing when logged in', () => {
  const component = shallow(<Homepage {...props} isLoggedIn />);
  expect(toJson(component)).toMatchSnapshot();
});

it('renders without crashing when not logged in', () => {
  const component = shallow(<Homepage {...props} />);
  expect(toJson(component)).toMatchSnapshot();
});
