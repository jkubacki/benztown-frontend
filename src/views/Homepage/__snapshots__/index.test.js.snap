// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing when logged in 1`] = `
<Fragment>
  <Connect(ElementList) />
</Fragment>
`;

exports[`renders without crashing when not logged in 1`] = `
<Fragment>
  <Connect(ReduxForm) />
</Fragment>
`;
